tests/test_pytest_cov.py

```diff
@@ -111,7 +117,11 @@ def test_fail():
 CHILD_SCRIPT_RESULT = '6 * 100%'
 PARENT_SCRIPT_RESULT = '8 * 100%'
 
-xdist = pytest.mark.parametrize('opts', ['', '-n 1'], ids=['nodist', 'xdist'])
+try:
+    import xdist
+    xdist = pytest.mark.parametrize('opts', ['', '-n 1'], ids=['nodist', 'xdist'])
+except:
```

Comment:
Why a bare `except:` here? That's always a bad idea...

Reply:
instead then use `xdist = pytest.mark.parameterize('opts', ['', requires_xdist('-n 1')], ids=['nodist', 'xdist'])` for old pytest

or `xdist = pytest.mark.parameterize('opts', ['', pytest.param('-n 1', marks=requuires_xdist)], ids=['nodist', 'xdist'])`